tests/fixtures/generated_input_data_and_results.py

```diff
@@ -7,7 +7,7 @@
 import os
 import numpy as np
 
-from gpucsl.pc.pc import DataDistribution, pc
+from gpucsl.pc.pc import GaussianPC
```

Comment:
Correct me if I'm wrong:
does the refactoring in this PR mean, that we no more have the unified `pc` interface for both Gaussian and discrete distributions? Should the benchmarks (e. g. `benchmark_gpucsl.py`)/ `README` / our report be adjusted as well?

Looks to me like quite a lot of last-minute changes. Do you think, it still can be done in time?

Reply:
does the refactoring in this PR mean, that we no more have the unified pc interface for both Gaussian and discrete distributions?

-> yes it does. We now have seperate PC classes for every kind of kind of CI test we support (although they all share the same constructor by design and then use a seperate function to set the distribution specific paramters, so we still have an uniform interface on one level. We just do not have this one pc function, that collects parameters for every distribution it supports, anymore)

Reply:
- change benchmarks and README -> yes. Until now I just implemented the core logic and made the tests run
- change the report -> I do not think this is necessary. The one sentence I spotted  in the architecture, that touches this topic is written in such a manner that I do not think it needs to be changed

Reply:
README and benchmarks are already changes. Just the documentation is left